### ISSUE TITLE
[SECURITY] path-parse 1.0.6 vulnerability in npm 6.14.13

### ISSUE THREAD

**Issue by networkandcode** [npm/cli#3317]
We are using the version of NPM that's installed via Node12, and then upgraded that to 6.14.13.

Is there a patch for this vulnerability

High CVSS  7.5	CVE-2021-23343	path-parse	1.0.6	fixed in 1.0.7	
All versions of package path-parse are vulnerable to Regular Expression Denial of Service (ReDoS) via splitDeviceRe, splitTailRe, and splitPathRe regular expressions. ReDoS exhibits polynomial worst-case time complexity.

```
├─┬ npm@6.14.13
│ └─┬ normalize-package-data@2.5.0
│   └─┬ resolve@1.10.0
│     └── path-parse@1.0.6
```

**Comment by woswos2018** [comment]
```
└─┬ npm@7.15.0
  └─┬ libnpmpublish@4.0.1
    └─┬ normalize-package-data@3.0.2
      └─┬ resolve@1.20.0
        └── path-parse@1.0.6
```

Could you please include path-parse 1.0.7 in the latest npm 7.x too?

**Comment by darcyclarke** [comment]
**Closing** This should be patched/resolved in [`v6.14.15`](https://github.com/npm/cli/releases/tag/v6.14.15)
